include/uv/win.h

```diff
@@ -86,6 +86,14 @@ typedef struct pollfd {
 #define SIGKILL               9
 #define SIGWINCH             28
 
+/* Redefine NSIG to take SIGWINCH into consideration */
+#if defined(NSIG) && NSIG < 29
```

Comment:
Can we use `SIGWINCH` instead of a magic constant 29?

Reply:
Yeah, you're right. Updated the PR to include the requested change.